quiz.sql

```diff
@@ -6,38 +6,43 @@ CREATE DATABASE westeros;
 --   `houses` has the following columns: id (serial primary key), name (text), motto (text), and sigil (text)
 --   `people` has an id ( serial primary key), name (text), title (text), quote (text), image (text), and age (integer)
 
--- your code starts here
+CREATE TABLE houses (id SERIAL PRIMARY KEY, name TEXT, motto TEXT, sigil TEXT);
 
--- your code ends here
+CREATE TABLE people (id SERIAL PRIMARY KEY, name TEXT, title TEXT, quote TEXT, image TEXT, age INTEGER);
 
 -- Question 2. Load the data from `westeros/houses.csv` into the `houses` table and the data from `westeros/people.csv` into the `people` table.
 
--- your code starts here
+\copy houses(name, motto, sigil) FROM './westeros/houses.csv' WITH (FORMAT csv, HEADER true)
 
--- your code ends here
+\copy people(name, title, quote, image, age) FROM './westeros/people.csv' WITH (FORMAT csv, HEADER true)
 
 -- Question 3. Insert `Cersei Baratheon,King Mother and Lady of Casterly Rock,"The gods have no mercy, that’s why they’re gods.",http://i.imgur.com/jYeeCJ3.jpg,40`
 -- into the people table, and `House Baratheon,Ours is the fury.,http://i.imgur.com/iJtBKIR.png`
 -- into the houses table
 
--- your code starts here
+INSERT INTO people(name, title, quote, image, age) VALUES('Cersei Baratheon', 'King Mother and Lady of Casterly Rock', 'The gods have no mercy, that’s why they’re gods.', 'http://i.imgur.com/jYeeCJ3.jpg', 40);
 
--- your code ends here
+INSERT INTO houses(name, motto, sigil) VALUES('House Baratheon', 'Ours is the fury.', 'http://i.imgur.com/iJtBKIR.png');
 
 -- Question 4. Write a query to get all attributes of people greater than a certain age *or* having a specific title.
 
--- your code starts here
+SELECT *
+FROM people
+WHERE age > 20
+ORDER BY age;
```

Comment:
This searches strictly for rows with an `age` attribute greater than 20, and does nothing with `title`.